src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -411,4 +411,18 @@ def word_break(string, length)
     # adds a <wbr> tag after an amount of given characters
     safe_join(string.scan(/.{1,#{length}}/), "<wbr>".html_safe)
   end
+
+  def toggle_sliced_text(text, slice_length = 50, id = nil)
+    return text if text.length < slice_length
+    id ||= "toggle_sliced_text_#{rand(9**10)}"
```

Comment:
@evanrolfe 
> maybe use a timestamp instead? thats how railscasts do it

although railscast is using seconds precision, doubt it is enough here, but with milliseconds it should work